examples/globe.js

```diff
@@ -3,26 +3,52 @@
 
 // Define initial camera position
 var positionOnGlobe = { longitude: 2.351323, latitude: 48.856712, altitude: 25000000 };
+var promises = [];
 
 // `viewerDiv` will contain iTowns' rendering area (`<canvas>`)
 var viewerDiv = document.getElementById('viewerDiv');
+var miniDiv = document.getElementById('miniDiv');
 
 // Instanciate iTowns GlobeView*
 var globeView = new itowns.GlobeView(viewerDiv, positionOnGlobe, { renderer: renderer });
+var miniView = new itowns.GlobeView(miniDiv, positionOnGlobe, {
+    maxSubdivisionLevel: 2,
+    noControls: true,
+    alphaClearValue: 0,
+});
 
-var promises = [];
+var minDistance = 10000000;
+var maxDistance = 30000000;
+
+miniView.mainLoop.gfxEngine.renderer.setClearColor(0xff0000, 0.0);
+
+globeView.onAfterRender = function onAfterRender() {
+    var distanceCamera = globeView.camera.camera3D.position.length();
+    var distance = Math.min(Math.max(distanceCamera * 1.5, minDistance), maxDistance);
+    var camera = miniView.camera.camera3D;
+    camera.position.copy(globeView.controls.moveTarget()).setLength(distance);
+    camera.lookAt(globeView.controls.moveTarget());
+    miniView.notifyChange(true);
+};
 
 function addLayerCb(layer) {
     return globeView.addLayer(layer);
 }
+
+function addLayerCbM(layer) {
+    return miniView.addLayer(layer);
+}
 // Add one imagery layer to the scene
 // This layer is defined in a json file but it could be defined as a plain js
 // object. See Layer* for more info.
 promises.push(itowns.Fetcher.json('./layers/JSONLayers/Ortho.json').then(addLayerCb));
+promises.push(itowns.Fetcher.json('./layers/JSONLayers/Ortho.json').then(addLayerCbM));
 // Add two elevation layers.
 // These will deform iTowns globe geometry to represent terrain elevation.
 promises.push(itowns.Fetcher.json('./layers/JSONLayers/WORLD_DTM.json').then(addLayerCb));
 promises.push(itowns.Fetcher.json('./layers/JSONLayers/IGN_MNT_HIGHRES.json').then(addLayerCb));
 
+globeView.miniview = miniView;
```

Comment:
If it's not used in the `globe_test.js` file then you don't need to export this.